apps/backend/api/graphql/mutations/membership.js

```diff
@@ -1,9 +1,11 @@
+/* global GroupMembership, GroupJoinQuestionAnswer, Queue, bookshelf */
```

Comment:
?? still want this?

Reply:
this is only needed for linting when looking at the code from the root of the monorepo. so I think its worth leaving in?

I did this from the mono repo because I had AI basically do the whole first pass, backend and front-end at once (I started it a bit to give it some help)! It did a pretty good job and I mostly vibe coded the rest, only a few tweaks needed.